packages/internal/src/build/babel/common.ts

```diff
@@ -30,3 +32,47 @@ interface BabelRegisterOptions extends TransformOptions {
 export const registerBabel = (options: BabelRegisterOptions) => {
   require('@babel/register')(options)
 }
+
+export const CORE_JS_VERSION = pkgJson.dependencies['core-js']
+  .split('.')
+  .slice(0, 2)
+  .join('.') // Produces: 3.12, instead of 3.12.1
+
+if (!CORE_JS_VERSION) {
+  throw new Error(
+    'RedwoodJS Project Babel: Could not determine core-js version.'
+  )
+}
+
+const RUNTIME_CORE_JS_VERSION = pkgJson.dependencies['@babel/runtime-corejs3']
+if (!RUNTIME_CORE_JS_VERSION) {
+  throw new Error(
+    'RedwoodJS Project Babel: Could not determine core-js runtime version'
+  )
+}
```

Comment:
I usually try to avoid constants like these because they execute when they're imported. These should not be slow, but over time if you accumulate a bunch of these and it could add up. 

My preferred way of doing this is just to stick them in functions, and do the computation when it's required.

Just a thought, nothing actionable.

Reply:
Sure, another case of me just copying things across, haven't put much thought into it!